random/string.go

```diff
@@ -0,0 +1,9 @@
+package random
+
+import "math/rand"
+
+func NewString(rng *rand.Rand, minLen, maxLenDiff int) string {
```

Comment:
Maybe add a bool for whether the string should be readable/printable?

Reply:
I am :thinking: Is there ever a reason one would not want a readable/printable `string`. Otherwise one could just use something like `string(random.NewBytes)`. Personally, I would just adapt this `NewString` implementation to always be readable.

Reply:
I agree, always make it readable, otherwise just take random `byte`s.

Reply:
I'd actually even simplify this to
```suggestion
func String(rng *rand.Rand, size int) string {
```
The caller can create the size randomly itself, if needed. Keeps the API clean, separation of concerns.

Note that I also propose to omit `New` so it reads nicely `random.String`. It's clear that it's a _new_ one IMO, since this new `random` package has this exact purpose.

Reply:
True!